homeassistant/components/html5/strings.json

```diff
@@ -40,6 +40,28 @@
           "description": "Extended information of notification. Supports tag."
         }
       }
+    },
+    "notify": {
```

Comment:
This definition would be for the `html5.notify` service, not `notify.html5` service.

As far as I'm aware, `html5.notify` doesn't exist.

../Frenck